usecase/usecase-webapi-xwalk-tests/steps/IAPGooglePlay/step.js

```diff
@@ -0,0 +1,10 @@
+var step = '<font class="fontSize">'
+            + '<p>Test Purpose: </p>'
+            + '<p>Verify to query the product details and buy the in-app product successfully.</p>'
+            + '<p>Expected Result: </p>'
+            + '<p>It will firstly show "Initialization succeeded."</p>'
+            + '<p>Querying the product details and buying the in-app product successfully.</p>'
+            + '<p>There is a new history named "Crosswalk IAP demo - Test Apple (Crosswalk IAP Demo)".</p>'
```

Comment:
It should be the latest purchase list, and you should tell people where to check this purchase list.